.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### PR Checklist
+- [ ] Have you updated the required documentation based on the current PR
+  changes
+- [ ] Have you written new tests for your core changes, as applicable?
+
+**Commit Formatting**
+
+Please follow the `seven rules of a great commit message`
+
+- Separate subject from body with a blank line
+- Limit the subject line to 50 characters
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Use the imperative mood in the subject line
+- Wrap the body at 72 characters
+- Use the body to explain what and why vs. how
+
+**Closing issues**
+
+Put Closes #XXXX`in your commit comment to auto-close this PR when merged.
```

Comment:
Missing a leading accent mark prior to `Closes`.  This implies that the commit requires amending otherwise you don't know the PR number.  Is this statement only necessary in the PR description or the commit comment itself?